docs/index.rst

```diff
@@ -15,11 +15,12 @@
 Welcome to twine's documentation!
 =================================
 
-Twine is `a utility`_ for `publishing`_ Python packages on `PyPI`_.
+.. include:: ../README.rst
+   :end-before: docs-badges-end
 
-It provides build system independent uploads of source and binary
-`distribution artifacts <distributions_>`_ for both new and existing
-`projects`_.
+.. include:: ../README.rst
+   :start-after: docs-intro-start
+   :end-before: docs-intro-end
```

Comment:
This makes `tox -e watch-docs` (i.e. `sphinx-autobuild`) slightly less useful, because the preview no longer updates when `README.rst` is changed.

Reply:
Hm... This is weird. Sounds like it's a bug in sphinx-autobuild.